testflags/testflags.go

```diff
@@ -0,0 +1,69 @@
+// The MIT License (MIT)
+
+// Copyright (c) 2017-2020 Uber Technologies Inc.
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package testflags
+
+import (
+	"flag"
+	"testing"
+)
+
+var TestFlags struct {
+	mysql     bool
+	postgres  bool
+	mongodb   bool
+	cassandra bool
+}
+
+func init() {
+	flag.BoolVar(&TestFlags.mysql, "mysql", false, "MySQL external dependency")
+	flag.BoolVar(&TestFlags.postgres, "postgres", false, "PostGreSQL external dependency")
+	flag.BoolVar(&TestFlags.mongodb, "mongodb", false, "MongoDB external dependency")
+	flag.BoolVar(&TestFlags.cassandra, "cassandra", false, "Cassandra external dependency")
+}
+
+// TODO: Better instructions:
+// 1) How to start external dependencies - maybe link to docs so we can change it easily?
```

Comment:
probably going to be something like "start cassandra (very likely `docker run cassandra:version`), and install the schema".
but that can come later.  and there's a million acceptable ways to do it, so deciding what's best may take a few rounds.